cumulusci/robotframework/javascript/cci_init.js

```diff
@@ -0,0 +1,38 @@
+/* This is passed to the Browser library via the jsextension
+   parameter when the library is imported.
+
+   This file will import all modules in the robot/<project
+   name>/javascript folder. The code in this file is dependent on an
+   environment variable named CCI_CONTEXT, which is initialized by the
+   robot task.
+*/
+
+const fs = require("fs");
+
+var cumulusci = require("cumulusci");
+var cci_context = JSON.parse(process.env.CCI_CONTEXT);
+cumulusci.project_config = cci_context.project_config;
+cumulusci.org = cci_context.org;
+
+// This is where each project should store their keywords.
+// Maybe in the future this should be configurable, but for
+// now it's not.
+var javascript_dir =
+    cci_context.project_config.repo_root +
+    "/robot/" +
+    cci_context.project_config.repo_name +
+    "/javascript";
```

Comment:
```suggestion
let javascript_dir = `${cci_context.project_config.repo_root}/robot/${cci_context.project_config.repo_name}/javascript`;
```

Up for debate if you find this clearer.